pennylane_lightning/core/src/bindings/Bindings.hpp

```diff
@@ -596,7 +636,11 @@ template <class StateVectorT> void lightningClassBindings(py::module_ &m) {
     auto pyclass_measurements = py::class_<Measurements<StateVectorT>>(
         m, class_name.c_str(), py::module_local());
 
+#ifdef _ENABLE_PLGPU
+    pyclass_measurements.def(py::init<StateVectorT &>());
+#else
     pyclass_measurements.def(py::init<const StateVectorT &>());
+#endif
```

Comment:
Why the disparity here?

Reply:
Aha, nice spot! I planned to try if `const` work with `cusparseCreateConstDnVec` after I finished CI workflows work. `const` doesn't work here for the moment is because the dense vector to be passed to SparseMV API is created by `cusparseCreateDnVec` now. `cusparseCreateDnVec` only works for `non const` pointer. Will keep you posted once I tried the `cusparseCreateConstDnVec`.

Reply:
Done!